repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/framework/mocks/MockValkyrie.groovy

```diff
@@ -76,10 +76,12 @@ class MockValkyrie {
     String account_perm = "also_butts"
     String contact_id = ""
     String tenant_id = ""
+    Integer fixnumber = 10
+    Integer multiplier = 1
 
```

Comment:
I'm confused.  To me, fixnumber and multiplier imply that at some point you will multiply the fixed number "fixnumber" by "multiplier" which means you could only ever have some number that's a multiple of the fixed number (10 in this case).  That's not what I'm seeing.  Like Bill mentioned, it seems like "fixnumber" really means "number of devices in the effective call" and "multiplier" really means "number of devices in the inventory call".